test/unit/test_task.rb

```diff
@@ -10,10 +10,12 @@
 require_relative 'tools/punchlist'
 require_relative 'tools/brakeman'
 require_relative 'tools/rails_best_practices'
+require_relative 'tools/eslint'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.